cluster-provision/centos8/scripts/dnsmasq.sh

```diff
@@ -6,8 +6,8 @@ NUM_NODES=${NUM_NODES-1}
 NUM_SECONDARY_NICS=${NUM_SECONDARY_NICS:-0}
 
 ip link add br0 type bridge
-echo 0 > /proc/sys/net/ipv6/conf/br0/disable_ipv6
-echo 1 > /proc/sys/net/ipv6/conf/all/forwarding
+echo 0 > /proc/sys/net/ipv6/conf/br0/disable_ipv6 || true
+echo 1 > /proc/sys/net/ipv6/conf/all/forwarding || true
```

Comment:
Did you check it with the latest providers ?
The nodes wait for ipv6 address by dnsmasq, so this fix might not be enough

(and also we concluded that we enforce even ipv6 but this is other story, see please the other comment about it)